app/views/play/cinematic/Command/CinematicParser.js

```diff
@@ -1,8 +1,40 @@
-import { getLeftCharacterThangTypeSlug, getRightCharacterThangTypeSlug } from '../../../../schemas/selectors/cinematic'
+import AbstractCommand from './AbstractCommand'
```

Comment:
This file has logic changes.
We iterate through systems checking if they have particular setup methods.
If they do we give them cinematic data and concatenate a list of commands returned from all systems.